src/activations.jl

```diff
@@ -357,6 +357,39 @@ function deriv_gelu(x)
     muladd(dσ * λλ * muladd(x2, α2, t), x, Ω)
 end
 
+"""
+    gelu_erf(x) = xΦ(x) = 0.5x * (1 + erf(x/√2))
+
+Activation function from ["Gaussian Error Linear Units"](https://arxiv.org/abs/1606.08415) without approximation.
+"""
+gelu_erf(x) = x/2*(1 + _erf(x/sqrt(oftf(x,2))))
+
+function deriv_gelu_erf(x)
+    SQRT2 = sqrt(oftf(x,2))
+    Φ = (1 + _erf(x/SQRT2))/2
+    Φ + x/SQRT2*exp(-(x^2)/2)/sqrt(oftf(x,π))
+end
+
+_erf(x::Number) = _erf(float(x))
```

Comment:
This intends to catch integers but isn't a safe pattern, as there are other weird numbers out there:
```
StackOverflowError:
  Stacktrace:
   [1] _erf(x::ForwardDiff.Dual{ForwardDiff.Tag{Zygote.var"#141#142"{var"#170#178"{typeof(gelu_erf)}}, Float64}, Float64, 1}) (repeats 79984 times)
     @ NNlib ~/work/NNlib.jl/NNlib.jl/src/activations.jl:373
```
This particular case could be allowed via NNlibForwardDiffExt